### PR TITLE
feat(kai-kli): add storage open command

### PR DESCRIPTION
**Pull request by JYisus**
## Description

- Refactor urls management
- add storage open command

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api/graphql/server_client.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func (g *GqlManager) setupClient(server *configuration.Server, args ...graphql.C
 	opts := []graphql.ClientOption{graphql.WithHTTPClient(c)}
 	opts = append(opts, args...)
 
-	g.client = graphql.NewClient(fmt.Sprintf("%s/graphql", server.URL), opts...)
+	g.client = graphql.NewClient(fmt.Sprintf("%s/graphql", server.APIURL), opts...)
```
Comment (kafkaphoenix) [inline api/graphql/server_client.go:106]: maybe we can rename this to API_URL?

**Review by kafkaphoenix — APPROVED** [review]

File: api/graphql/server_client.go (hunk 1 above)
Comment (JYisus) [reply api/graphql/server_client.go:106]: It's no a good practice in Golang use underscores in names, We could use ApiURL maybe

Comment (JYisus) [reply api/graphql/server_client.go:106]: Well, the linter said no 
var-naming: struct field ApiURL should be APIURL (revive)

**Review by dfinteligenz — APPROVED** [review]

File: api/graphql/server_client.go (hunk 1 above)
Comment (kafkaphoenix) [reply api/graphql/server_client.go]: okey fine then

**Review by kafkaphoenix — APPROVED** [review]

**Pull request merged**